src/main/java/org/apache/commons/text/CaseUtils.java

```diff
@@ -32,6 +32,8 @@
  * @since 1.2
  */
 public class CaseUtils {
+    public static final int SPACE_CODEPOINT = 32;
```

Comment:
Hello @ikaronen-relex 
Thank you for your PR.
This constant does not need to be part of the public API.
Shouldn't the set contain chars from `Character.isSpaceChar()` and/or `Charcter.isWhitespace()`?

Reply:
Thanks! I fixed the visibility.

It might indeed make sense for other whitespace characters to be part of the default delimiter set, but not knowing how this class may be used, I feel a bit hesitant changing its behavior so much.

Reply:
Hello @ikaronen-relex and thank you for your reply.

OK, that makes sense, now all you have to do is make sure it compiles ;-) Note that the project requirement in pom.xml is Java 8.